web-console/4.0-designs/storage/Clone/clone.md

```diff
@@ -0,0 +1,55 @@
+# Clone - PVC/PV
+
+## Main use cases
+Create clone (from where PVC/snapshot is)
+All other List, View, Delete, Modify actions are that of a PV/PVC
+
+#### Notes:
+- Alpha support in Kubernetes v1.15
+- critical for stateful workloads, especially databases
+- similar behavior to creating a snapshot
+- From user’s perspective, a clone is another PVC/PV; however contents of a clone is that its PV is populated from contents of a PV at creation time.
+- After creation, the clone behaves just like any other PV.
```

Comment:
should this read "any other PVC/PV" ? It sounds like here when you clone a PVC it results in a PV, is that true?

Reply:
@matthewcarleton When you clone a PVC (which would have a backing PV), it will result in another PVC being created (along with a backing PV).  The new PVC created will be populated using the contents of the source backing PV at creation time.

Reply:
@julienlim - How does that work? The original PVC is bound to a PV I'm assuming, would the newly created PVC via clone just sit there unbound?

Reply:
@lizsurette The newly created PVC (clone) will dynamically create a matching PV to back it.  You will not be able to create a PVC completely if there is no backing PV and underlying backing storage.  As previously noted, a clone works pretty much like a PVC/PV.  There is only 1 piece of metadata that is changed in the YML which basically indicates the source when you create it via YML.  Other than that, it's really the same as a PVC/PV.